PurchasesTests/Networking/BackendTests.swift

```diff
@@ -523,7 +523,14 @@ class BackendTests: XCTestCase {
         })
 
         expect(error).toEventuallyNot(beNil())
-        expect(error?.code).toEventually(be(ErrorCode.invalidCredentialsError.rawValue))
+        let attachedErrorKey = ErrorDetails.attachedErrorKey as String
+        let unexpectedBackendError = error?.userInfo[attachedErrorKey] as? NSError
+        let customerInfoError = unexpectedBackendError?.userInfo[attachedErrorKey] as! CustomerInfoError
+        let extraContextString = unexpectedBackendError?.userInfo[ErrorDetails.extraContextKey as String] as! String
+
+        expect(extraContextString).to(equal("code=7225,message=something is bad up in the cloud"))
+        expect(customerInfoError.rawValue).to(equal(CustomerInfoError.missingJsonObject.rawValue))
+        expect(error?.code).toEventually(equal(ErrorCode.invalidCredentialsError.rawValue))
```

Comment:
✅ left `toEventually` the same, though.

Reply:
ok! I actually don't think it's needed since it should wait in like 525 right? I also don't know how it works so feel free to ignore lol. But it was like this before so leave it like that

Reply:
Oohhh I totally missed that, ya, you're right!